src/ac_copilot_trainer/modules/coaching_overlay.lua

```
@@ -80,26 +88,129 @@ function M.draw(coachingLines, timeRemaining, holdSeconds)
+  local hold = vm.coachingHoldSeconds or 30
+  local primer = vm.coachingShowPrimer
+
+  local showActive = lines and #lines > 0 and rem and rem > 0
+  local showPrimerBand = primer and not showActive
+  if not showActive and not showPrimerBand then
+    return
+  end
+
+  local alpha = 1.0
+  local title = "COACHING"
+  local body = ""
```

Luacheck (1.2.0)

[warning] value assigned to variable 'body' is unused

(W311)